hoprd/rest-api/src/lib.rs

```diff
@@ -164,16 +173,27 @@ impl Modify for SecurityAddon {
     }
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
Would a better approach be to create an intermediate input object?

Reply:
Or maybe a builder

Reply:
We can tackle this in a separate refactoring PR.